src/test/scala/com/microsoft/hyperspace/TestUtils.scala

```diff
@@ -107,3 +107,15 @@ object MockEventLogger {
     emittedEvents = Seq()
   }
 }
+
+object TestConfig {
+  val HybridScanEnabled = Seq(
+    IndexConstants.INDEX_HYBRID_SCAN_ENABLED -> "true",
+    IndexConstants.INDEX_HYBRID_SCAN_APPENDED_RATIO_THRESHOLD -> "0.99",
+    IndexConstants.INDEX_HYBRID_SCAN_DELETED_RATIO_THRESHOLD -> "0.99")
```

Comment:
Can we use "1.0" instead? (not sure what it means by "0.99")

Reply:
Are the tests failing if we use the default ratio? I am a bit confused with when to use this config, looking at other tests below.

Reply:
Yes because the sample data is too small, so almost all tests failed with the default config. 
"1.0" means Hybrid scan is performed even if there's no common source data, so I set it "0.9" or "0.99"